patches/win32-dll-build.patch

```diff
@@ -0,0 +1,38 @@
+From c3236b6df9af74ec4c0c5aa9d5b1d95b43392a4a Mon Sep 17 00:00:00 2001
+From: Kartik Singh <kssingh@hrl.com>
+Date: Thu, 12 Dec 2024 11:40:25 -0800
+Subject: [PATCH] Patch libsbcl build for Windows
+
+---
+ make-shared-library.sh  | 2 +-
+ src/runtime/GNUmakefile | 4 ++--
+ 2 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/make-shared-library.sh b/make-shared-library.sh
+index 9acd9d843..39a951e9e 100755
+--- a/make-shared-library.sh
++++ b/make-shared-library.sh
+@@ -5,4 +5,4 @@
+ echo //entering make-shared-library.sh
+ echo //building sbcl runtime into a shared library
+ 
+-$GNUMAKE -C src/runtime libsbcl.so
++$GNUMAKE -C src/runtime libsbcl.dll
+diff --git a/src/runtime/GNUmakefile b/src/runtime/GNUmakefile
+index 15e12a995..59d61592e 100644
+--- a/src/runtime/GNUmakefile
++++ b/src/runtime/GNUmakefile
+@@ -132,8 +132,8 @@ libsbcl.a: $(OBJS)
+ 	rm -f $@ ; ar rcs $@ $^
+ 
+ PIC_OBJS = $(subst .o,.pic.o,$(OBJS))
+-libsbcl.so: $(PIC_OBJS)
+-	$(CC) -shared -o $@ $^ $(LIBS) $(SOFLAGS)
++libsbcl.dll: $(PIC_OBJS)
++	$(CC) -shared -o $@ $^ $(LIBS) $(SOFLAGS) -Wl,-export-all-symbols -Wl,--whole-archive -lmpfr -Wl,--no-whole-archive mswin64.def
```

Comment:
Insert note in modified code regarding linkage of log symbols in mpfr; possibly, note about windows sensitivity to file extension